client/src/pages/view-my-claims.component.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import MyClaimFeed from '../components/myClaimFeed.component';
+
+export default class ViewMyClaims extends Component {
+  render() {
+    return (
+      <>
+        <div>Your claimed listings</div>
```

Comment:
We probably have this in a few pages/components already and can save correction for post-freeze tidying, but the semantic meaning of a div is simply 'a division of the page'/a section/a container and shouldn't have text in it without additional markup. Text should all be in a header/paragraph/caption, etc. For screen readers and also search engine optimisation etc.